packages/shorebird_cli/lib/src/artifact_manager.dart

```diff
@@ -1,3 +1,4 @@
+// cspell:words archs xcarchive xcframework
```

Comment:
Maybe this should be configured globally on cspell?

Reply:
Yeah, we need to figure out our cspell settings.  Some of these definitely belong global.